src/config.rs

```diff
@@ -14,7 +14,7 @@ use {NewlineStyle, BraceStyle, ReturnIndent};
 use lists::SeparatorTactic;
 use issues::ReportTactic;
 
-#[derive(RustcDecodable)]
+#[derive(RustcDecodable, Clone, Copy)]
```

Comment:
This shouldn't be `Copy` - it might get more complex in the future and is already too big to morally be copy-able